packages/terra-heading/src/terra-dev-site/doc/example/HeadingLevels.jsx

```diff
@@ -8,6 +8,15 @@ const cx = classNames.bind(styles);
 
 const HeadingLevels = () => (
   <div>
+    <p>
+      Terra includes styling for all standard headings `h1` through `h6`, as well as styles that match the size of their respective heading. Headings should be
+      marked up according to the document hierarchy, not appearance. To facilitate navigation and understanding of overall document structure, authors should use
+      headings that are properly nested (e.g., `h1` followed by `h2`, `h2` followed by `h2` or `h3`, `h3` followed by `h3` or `h4`, etc.). The `level` prop accepts
+      one of the following numbers:
+      <br />
+      <br />
+      `1`, `2`, `3`, `4`, `5`, `6`
+    </p>
```

Comment:
These look weird
![Screen Shot 2020-03-18 at 3 04 55 PM](https://user-images.githubusercontent.com/14099737/77002672-e08af480-6929-11ea-8afc-2d79b230e991.png)

Reply:
It looks weird but when we update the example loader to handle highlight quotes. This would look similar to that on the current dev-site.

Reply:
Don't put documentation inside the examples, move this it the MDX documentation above the example.